docs/changes/tests/17883.api.rst

```diff
@@ -0,0 +1 @@
+Removed ``astropy.tests.command`` module that was deprecated in v6.0. In addition, ``astropy.test``, ``astropy.tests.runner.TestRunnerBase``, and ``astropy.tests.runner.TestRunner`` are now officially deprecated. This will also affect downstream ``packagename.test`` generated using ``TestRunner``. The deprecated functionality will be removed in a future release after a deprecation period as per Astropy deprecation policy.
```

Comment:
I think these should be two separated entries, it would be a better help to find the relevant entries when the removals are separated out from the deprecations.

Reply:
Nope, didn't work. Let me try something else.

Reply:
Or I could just bug Stuart with this, I'm about to meet him 😂